modules/azure-oci-zone-mapping/main.tf

```diff
@@ -0,0 +1,79 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+  }
+}
+
+
+## uncomment for local module run
+# provider "azurerm" {
+#   features {}
+# }
+
+#https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/location
+data "azurerm_location" "azure_location_info" {
+  location = var.location
+}
+
+locals {
+  zone_mapping = [for zone_mapping in data.azurerm_location.azure_location_info.zone_mappings : zone_mapping
+    if zone_mapping.logical_zone == var.zone
+  ]
+  azure_to_oci_ad_internal_mappping = tomap({
```

Comment:
Can we use json for this mapping and import into this module, instead of having the mapping here